Race/race.py

```diff
@@ -0,0 +1,75 @@
+import random
+import time
+
+
+class Animal:
+
+    def generatenumber(self):
+        num = random.randint(1, 10)
+        return num
+
+    def move(self):
+        return self.generatenumber()
+
+
+class TortoiseClass(Animal):
+    def move(self,tortoise):
+        tnum = super().move()
+        if tnum in range(1, 6):
+            tortoise = int(tortoise + 3)
```

Comment:
let's use `self.postion` field that you will define in animal base class which hold the current position of current animal object.